libraries/AP_Logger/LogStructure.h

```diff
@@ -1224,10 +1225,10 @@ struct PACKED log_Arm_Disarm {
 #define MAG_UNITS "sGGGGGGGGG-s"
 #define MAG_MULTS "FCCCCCCCCC-F"
 
-#define PID_LABELS "TimeUS,Des,Act,P,I,D,FF"
-#define PID_FMT    "Qffffff"
-#define PID_UNITS  "s------"
-#define PID_MULTS  "F------"
+#define PID_LABELS "TimeUS,Des,Act,Err,P,I,D,FF"
```

Comment:
missing s/Des/Tar/ ?

Reply:
@amilcarlucas Sorry, what are you saying here?

Reply:
You renamed "desired" to "target" everywhere.
I assume you also want to change:

-#define PID_LABELS "TimeUS,Des,Act,Err,P,I,D,FF"
+#define PID_LABELS "TimeUS,Tar,Act,Err,P,I,D,FF"

Reply:
Ahh, bingo. Thanks!!